packages/cmrjs/utils.js

```diff
@@ -103,6 +103,16 @@ function ummVersion(umm) {
   return _get(umm, 'MetadataSpecification.Version', '1.4');
 }
 
+/**
+ * Transform UMM version number to metadata format string.
+ *
+ * @param {string} versionNumber - UMM version string in decimal format (e.g. 1.4)
+ * @returns {string} UMM-G metadata format string (e.g. umm_json_v1_4)
+ */
+function ummVersionToMetadataFormat(versionNumber, ummFormat = 'json') {
```

Comment:
I know this is used by the int tests, but we should have a few quick unit tests on this.